### DISCUSSION TITLE
Half pie chart

### DISCUSSION THREAD

**Discussion by maximerivolet** [airbnb/visx#1012] [Q&A] [answered]
Hi,

I am trying to draw a half pie chart. I tried using startAngle, set to -90 and endAngle, set to 90. I would like this kind of result :

https://datavizproject.com/wp-content/uploads/2016/06/DVP_1_100-76.png

Is it possible with Visx ?

Regards,

**Comment by hshoff** [comment, marked as answer]
Hi @goldenivan, yes this is possible but `startAngle` and `endAngle` [need to be radians](https://github.com/d3/d3-shape#arc_startAngle). 

```tsx
const HALF_PI = Math.PI / 2;

<Pie
  // ...
  startAngle={-HALF_PI}
  endAngle={HALF_PI}
/>
```

Here's a demo: https://codesandbox.io/s/eloquent-bush-gbvxi

![image](https://user-images.githubusercontent.com/339208/104198733-90b77580-53f4-11eb-8313-c9dc0a4f4548.png)

**Comment by maximerivolet** [comment]
Bonjour,

Merci pour le retour rapide ! C'est effectivement la solution.
